docs/components/motor/28byj48.md

```diff
@@ -22,6 +22,7 @@ Viam also supports:
 {{< /alert >}}
 
 To configure a `28byj48` motor as a component of your machine, first configure the [board](/components/board/) to which the motor driver is wired.
+Make sure to connect your motor to your board and turn it on.
 Then, add the motor:
```

Comment:
28byj48 is similar to gpio stepper motor, so same applies here.

Also I think especially for these longer paragraphs, but really everywhere, we should separate the intro sentence from these instructions with a line break.

```suggestion

To use a `28byj48` motor as a component of your machine, first wire your 28BYJ-48 or similar small stepper motor to a suitable motor driver such as the ULN2003, and wire the driver to a board.
Connect the system to power if you want to test it while configuring.
Configure the [board](/components/board/) to which the motor driver is wired.
Then configure the motor:
```